File: 백준/Silver/2622. 삼각형만들기/삼각형만들기.cc
#include <iostream>
using namespace std;
int main() {
    long long dap = 0;
    int n;
    cin>>n;
    for (int x =1 ; x<=n/3 ; x++){
        for (int y = 0 ; y < n; y++){
            int first=x;
            int second=x+y;
            int third=n-(first+second);
            if (second>third) break;
            if (first+second>third) dap++;
        }
    }
cout<<dap;
    
    return 0;
}

File: 백준/Silver/2622. 삼각형만들기/삼각형만들기.py
n=int(input())
dap=0
for x in range(1,n//3+1):
    for y in range(n):
        first=x
        second=x+y
        third=n-(first+second)
        if second<=third and first+second>third:
            dap+=1
        if second>third:
            break
print(dap)